RFC/0000-template.md

```diff
@@ -0,0 +1,63 @@
+# RFC 0000: Request for Change Template
+
+<!-- RFC number will be assigned by an editor. When opening a pull request to
+submit your RFC, please use number 0000 and an abbreviated title in the filename,
+`0000-draft-title-abbrev.md`. -->
+
+- Start Date: (format: YYYY-MM-DD)
+- Related Issues:
+  - (add links here)
+
+# Summary
+
+<!--One paragraph explanation of the RFC.-->
+This is the suggested template for new RFCs.
+
+# Motivation
+
+AKA Problem Statement
+
+Clearly explain why the existing protocol specification is inadequate
+to address the problem that the RFC solves.
+
+# Detailed design
+
+AKA Solution Proposal
+
+Describe the proposed solution and list all changes made to the specs repository.
+
+The resulting specification should be detailed enough to allow competing,
+interoperable implementations.
+
+## Test fixtures
+
+List relevant CIDs. Describe how implementations can use them to determine
+specification compliance.
+
```

Comment:
An RFC may be a bit too early for this.

Reply:
I've added a note that this section can be skipped if the improvement proposal does not deal with the way IPFS deals with content-addressed data, but believe **we should keep Test fixtures prompt in the template** to encourage people to provide test data – imo more than often, it will be possible, and will make discussion way easier.